requirements.txt

```diff
@@ -14,5 +14,7 @@ yamllint>=1.15.0
 cffi
 rfc3987==1.3.8
 GitPython==2.1.11
+hvac==0.9.2
+docker==4.0.2
```

Comment:
We need docker for?

Reply:
`docker` or `docker-py` is needed for testing the authentication and getting the secrets from Hashicorp Vault. Currently, I'm spinning up a docker container of Vault and initializing it with test policies, authentication methods, and secrets, then testing it.